src/UserService.Data.Provider.MsSql.Ef/Migrations/20210317144051_AddPropertyCreatedAt.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using LT.DigitalOffice.UserService.Models.Db;
+using LT.DigitalOffice.UserService.UserCredentials.Admin;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace LT.DigitalOffice.UserService.Data.Provider.MsSql.Ef.Migrations
+{
+    [DbContext(typeof(UserServiceDbContext))]
+    [Migration("20210317144051_AddPropertyCreatedAt")]
+    public class AddPropertyCreatedAt : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTime>(
+                name: nameof(DbUser.CreatedAt),
+                table: DbUser.TableName,
+                nullable: false,
+                defaultValue: DateTime.Now
+            );
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: nameof(DbUser.CreatedAt),
+                table: DbUser.TableName
+            );
+        }
+
+        protected override void BuildTargetModel(ModelBuilder modelBuilder)
```

Comment:
No need to create BuildTargetModel, think you should remove it

Reply:
done